Pipfile

```diff
@@ -15,3 +15,4 @@ flake8-docstrings = "*"
 pre-commit = "*"
 coveralls = "*"
 coverage = "*"
+ipykernel = "*"
```

Comment:
?

Reply:
Running jupyter notebook in vs-code

Reply:
By default I'd err towards removing it as a dependency but I don't feel super strongly

Reply:
Sounds good. We have it in almost all of our other projects too 😬 😅